### DISCUSSION TITLE
Chunk option to control number of decimals

### DISCUSSION THREAD

**Discussion by snhansen** [gadenbuie/epoxy#125] [Ideas]
Hi Garrick,

I've just started using epoxy when writing reports and I'm very much enjoying it. One thing that I feel is quite cumbersome, however, is the control of the number of digits/decimals in the numbers reported. Currently, whenever I report a number I'd have to do something like `{round(x, 2)}` or even `{format(round(x, 2), nsmall = 2)}` to get the desired output. Would it be an idea to have a chunk option to control this behavior once and for all? Or is there another way to do this smarter?

**Comment by snhansen** [comment]
Already possible using the `.transformer`-argument. Sorry.

**Reply by gadenbuie** [reply]
No worries! Not wanting to repeat all of those settings is actually the source of the idea behind the package!

For others, you can find more information about inline transformers here: https://pkg.garrickadenbuie.com/epoxy/reference/epoxy_transform_inline.html
